### PR TITLE
Update contentful-util.js

### PR DESCRIPTION
**Pull request by drkstr101**
Filter out nullish values

### CODE REVIEW

File: lib/contentful-util.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const { documentToHtmlString } = require('@contentful/rich-text-html-renderer');
 const pkg = require('../package.json');
 
 function normalizeEntries({ contentTypes, entries, defaultLocaleCode, options }) {
-    return entries.map(entry => normalizeEntry({ contentTypes, entry, defaultLocaleCode, options })).filter(entry => entry);
+    return entries.map(entry => normalizeEntry({ contentTypes, entry, defaultLocaleCode, options })).filter(Boolean).filter(entry => entry);
```
Comment (smnh) [inline lib/contentful-util.js:5]: maybe better to delete the `.filter(entry => entry)` in favor of `.filter(Boolean)` then?
I think the `.filter(entry => entry)` had the same intention.

**Review by smnh — CHANGES_REQUESTED** [review]

File: lib/contentful-util.js (hunk 1 above)
Comment (drkstr101) [reply lib/contentful-util.js:5]: Ah yes, I was wondering about this myself. I thought `.filter(entry => entry)` would have had the same effect, but it seems to be letting the nulls through. I wanted to do some more testing, but the filter(Boolean) seemed to do the trick for now.

**Comment by drkstr101** [comment]
Reference: https://www.notion.so/stackbit/d34d262143e74e25b3bd0f491d59b41e?v=4033e253257d443b875e30249b60528a&p=c2ade56811e44ee79cc445b7eae979a6

**Comment by drkstr101** [comment]
Created test project: https://github.com/drkstr101/contentful-theme-test

File: lib/contentful-util.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const { documentToHtmlString } = require('@contentful/rich-text-html-renderer');
 const pkg = require('../package.json');
 
 function normalizeEntries({ contentTypes, entries, defaultLocaleCode, options }) {
-    return entries.map(entry => normalizeEntry({ contentTypes, entry, defaultLocaleCode, options })).filter(entry => entry);
+    return entries.map(entry => normalizeEntry({ contentTypes, entry, defaultLocaleCode, options })).filter(Boolean);
```
Comment (VitaliyR) [inline lib/contentful-util.js:5]: Weird why it's helping as nulls are still filtered out with `entry => entry`
image

**Review by smnh — APPROVED** [review]

**Pull request merged**